pkg/controller/replicator.go

```diff
@@ -192,12 +192,15 @@ func (r *replicator) Snapshot(name string, userCreated bool, created string, lab
 		if backend.mode != types.ERR {
 			wg.Add(1)
 			go func(address string, backend types.Backend) {
+				log := logrus.WithFields(logrus.Fields{"snapshot": name, "address": address})
+				log.Info("Starting to snapshot")
 				if err := backend.Snapshot(name, userCreated, created, labels); err != nil {
```

Comment:
There is already a start log in `backend.Snapshot(name, userCreated, created, labels)`. We can improve the logs in the implementation of this interface instead:
1. Add the address in the log
2. Add finish log

The implementation is here:
https://github.com/longhorn/longhorn-engine/blob/0c9b45e92f4c4b5dfd7dae5067d017846dbdc724/pkg/backend/remote/remote.go#L85

Reply:
done.